src/OrchardCore.Modules/OrchardCore.AuditTrail/Services/DateTimeParser.cs

```diff
@@ -51,9 +51,9 @@ public Expression BuildOperation(in BuildExpressionContext context, ConstantExpr
         }
     }
 
-    public class DateNode : OperatorNode
+    public class DateTimeOffsetNode : OperatorNode
```

Comment:
Did you check if these don't break Audit Trail admin listing? Since these are indirectly used from `DefaultAuditTrailAdminListFilterProvider`.

Reply:
If we have a list of `OperatorNode` in a List that is serialized as Json then it will break.
And we (with Mike) don't remember configuring these for custom json serialization (might have missed it).

Reply:
If this could break anyone, I will close this then